pyClient/zethMimc.py

```diff
@@ -9,6 +9,7 @@
     from Crypto.Hash import keccak
     keccak_256 = lambda *args: keccak.new(*args, digest_bits=256)
 
+from zethConstants import ZETH_MIMC_PRIME
 
 ##### SOME HASHES
```

Comment:
Del line

Reply:
If I remove this line constructor of the class won't work since it takes prime as optional parameter